src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/books";
+    private static final String DRIVER = "com.mysql.cj.jdbc.Driver";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "password";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", USERNAME);
+        DB_PROPERTIES.put("password", PASSWORD);
```

Comment:
I would suggest to make 'user' and 'password' literals constants rather than it`s values, because values could change dynamically, but words 'user' and 'password' will remain same